src/main/java/com/exercise/sfe/configuration/DatasourceConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package com.exercise.sfe.configuration;
+
+import javax.sql.DataSource;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.transaction.TransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+
+@Configuration
+@PropertySource("classpath:datasource.properties")
+@EnableTransactionManagement
+@RequiredArgsConstructor
+public class DatasourceConfiguration {
+
+  private final Environment environment;
+
+  @Bean
+  public DataSource dataSource() {
+    var dataSource = new DriverManagerDataSource();
+    dataSource.setDriverClassName(environment.getProperty("datasource.driver-class-name"));
+    dataSource.setUrl(environment.getProperty("datasource.url"));
```

Comment:
You can move such properties (as "datasource.url", ".password", etc) to some UtilityClass